SDK/AppCenterDistribute/Microsoft.AppCenter.Distribute.Android/Distribute.cs

```diff
@@ -51,6 +51,30 @@ static void HandleUpdateAction(UpdateAction updateAction)
             }
         }
 
+        static void SetUpdateTrack(UpdateTrack updateTrack)
```

Comment:
We could just use int value of the enum, on Android it's just an int.

Reply:
We already have the same implementation [here](https://github.com/microsoft/appcenter-sdk-dotnet/blob/bb4d3ed7ff3195f93397a8845210070e0cc83262/SDK/AppCenterDistribute/Microsoft.AppCenter.Distribute.Android/Distribute.cs#L39) for `HandleUpdateAction(UpdateAction updateAction)` method. I guess we should have a similar implementation here too.

Reply:
Yes we should, though it's out of scope for this PR or feature. Could be a bug.